### ISSUE TITLE
Log file retrieval with c8y_log_plugin fails to work after some time

### ISSUE THREAD

**Issue by bjoernsauer** [thin-edge/thin-edge.io#1688]
**Describe the bug**

Retrieving logs from the device via the c8y_log_plugin fails after some time.
If the device connects to the Cumulocity IoT cloud log retrieval works as expected. After some time the log retrieval does not work anymore. For us we recognized the error after 21h the next day but on a test device this also happened after about 1:15h.
In the cloud Dashboard the error reason is stated with:

```
Handling of operation failed with HTTP status client error (401 Unauthorized) for url (https://certification-phoenixcontact.eu-latest.cumulocity.com/event/events/) 
```

Output of the c8y_log_plugin.log file:

```
2023-01-19T13:36:19.62695126Z  INFO c8y_log_plugin: Log request received: 522,LaborDevice2152,mosquitto,2023-01-18T12:36:00+0100,2023-01-19T12:36:00+0100,,1000
2023-01-19T13:36:19.81214498Z ERROR c8y_log_plugin: Handling of operation: '522,LaborDevice2152,mosquitto,2023-01-18T12:36:00+0100,2023-01-19T12:36:00+0100,,1000' failed with HTTP status client error (401 Unauthorized) for url (https://certification-phoenixcontact.eu-latest.cumulocity.com/event/events/)
```

**To Reproduce**

Connect the device to the cloud wait some time and try to retrieve a log from the device.

**Expected behavior**

Log file retrieval should work also if the device is connected for longer time in idle mode.

**Environment (please complete the following information):**

 - Custom build yocto image
 - Phoenix Contact AXC F 2152 PLC
 - System-Architecture: Linux axcf2152 5.4.47-rt28-pxc SMP PREEMPT_RT Wed Dec 15 20:26:30 UTC 2021 armv7l armv7l armv7l GNU/Linux

 - thin-edge.io version 0.8.1

**Comment by reubenmiller** [comment]
Thanks @bjoernsauer for the info. This ticket is actually a duplicate of the same 401 problem that affects the configuration plugin (as the both use the same crate).

The bug fix has already been merged into `main` and will be part of the [0.9.0](https://github.com/thin-edge/thin-edge.io/milestone/18) release which we are hoping to deliver next week. You can click on the link to checkout the progress of the 0.9.0 release.

**Comment by reubenmiller** [comment]
thin-edge.io [0.9.0](https://github.com/thin-edge/thin-edge.io/releases/tag/0.9.0) has been released which includes the bug fix for this ticket 🍾
